docs/_v4-in-progress.mdx

```diff
@@ -0,0 +1,3 @@
+:::warning
+This page has not yet been updated for Helm 4. Some of the content might be inaccurate or not applicable to Helm 4. For more information about the Helm 4 new features, improvements, and breaking changes, see [Helm 4 Overview](/overview.md).
+:::
```

Comment:
Added this temporary warning to a partial that's then imported on the pages that need to be updated for Helm 4

Reply:
Nice solution. Importing one central warning like this feels better than copy/pasting the same static message into the (so far) 20 files identified that need the message. Updating the message, etc.

Related, but could be a follow-up issue: changing these files from `.md` to `.mdx` reminds me that we should make sure the Contributing doc has a note about CommonMark changes needed like escaping characters for `.mdx` files specifically (we use auto-detect so we don't need to do that on `.md` files). Can maybe just link to [markdown and JSX interoperability](https://docusaurus.io/docs/markdown-features/react#markdown-and-jsx-interoperability).